opencga-app/app/scripts/azure/arm/README.md

```diff
@@ -19,6 +19,177 @@ Click the following link the ensure you fill in the parameters according to thei
+
+Each of the web servers also run a health check to ensure key services are responding as expected. These can be viewed iva `loganalytics` as follows:
+
+```
+ContainerLog
+| where LogEntry contains "HealthCheck :"
+| extend failing = (LogEntry contains "KO")
+| project TimeGenerated, Computer, Name, LogEntry, failing 
+| order by TimeGenerated desc 
+```
+
+This returns the results from each node which look like `HealthCheck : {Solr=OK, CatalogMongoDB=OK, VariantStorage=OK, VariantStorageId=hadoop}`. If a failure has occurred the `OK` will be `KO`. For example, the following shows a failure talking to `hadoop`: `{Solr=OK, CatalogMongoDB=OK, VariantStorage=KO, VariantStorageId=hadoop}\n` 
+
+We can then use `loganalytics` to graph these failures over time grouped by each web server.
+
+```
+ContainerLog
+| where LogEntry contains "HealthCheck :"
+| extend failing = (LogEntry contains "KO")
+| project TimeGenerated, Computer, Name, LogEntry, failing 
+| summarize count(failing) by Computer, bin(TimeGenerated, 5m) 
+| render timechart 
+```
+
+In this example we see a failure occur around `10:20UTC` and the various web server nodes report their failures. 
+
+![healthcheck graph](docs/healthcheck.jpg)
+
+
+### Daemon VM checks and FAQ
+
+Q. How do I connect to the web servers hosting the `tomcat` app. 
+A. You can use the `daemonvm` as a jumpbox. Retreive the `daemonvm`'s public IP from the portal and use `ssh -J opencgaadmin@DAEMONVM_PUBLIC_IP_HERE opencgaadmin@opencga000001` to connect to the instances. You can change increment `opencga000001` to connect to other instances of the web servers.
+
+Q. What is the expected state of the `daemonvm` after deployment?
+A. `cloud-init status` should return a status of `done` and `sudo docker ps -a` should return 
+
+```
+CONTAINER ID        IMAGE                                 COMMAND                  CREATED             STATUS                             PORTS                                                    NAMES
+e948dabbc0ec        lawrencegripper/opencga-daemon:nfs1   "/bin/bash -c 'echo …"   19 minutes ago      Up 57 seconds (health: starting)                                                            opencga-daemon
+a37fce3165dd        lawrencegripper/opencga-init:nfs1     "/bin/bash /tmp/setu…"   20 minutes ago      Exited (0) 19 minutes ago                                                                   opencga-init
+2d00c901c3e9        microsoft/oms                         "/opt/main.sh"           23 minutes ago      Up 22 minutes                      127.0.0.1:25224->25224/udp, 127.0.0.1:25225->25225/tcp   omsagent
+
+```
+If the `opencga-init` container has an exit code other than `0` of the `opencga-daemon` `status` is `Restarting` then use `sudo docker logs opencga-init` or `sudo docker logs opencga-daemon` to review the logs.

```

Comment:
typo: `of the opencga-daemon status` should probably be `or the opencga-daemon status`